libs/designsystem/src/lib/components/action-sheet/action-sheet-popout/action-sheet-popout.component.html

```diff
@@ -7,9 +7,10 @@
     size="lg"
     attentionLevel="4"
     tabindex="-1"
+    [ngClass]="{ 'is-focused': focusedItemIndex == i }"
```

Comment:
Consider simplifying:
```suggestion
    [class.is-focused]="focusedItemIndex == i"
```